[Bug]: [Nightly] Nightly runs much slowly than ever and will be killed over the overall timeout

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:cbae252
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.0.dev26
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior


Nightly runs much slowly than ever and will be killed over the overall timeout


### Expected Behavior

Complete

### Steps To Reproduce

```markdown
nightly :
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Nightly-Debug/detail/test-binbin-nightly-debug/4/pipeline/155
```


### Milvus Log

1. log: [artifacts-milvus-distributed-kafka-nightly-4-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Nightly-Debug/job/test-binbin-nightly-debug/4/artifact/artifacts-milvus-distributed-kafka-nightly-4-pymilvus-e2e-logs.tar.gz)
2. some panic message file:
```
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-2494ddbfbfbd6ad237f1c55b70dea6f5b8dd90eb8dc25d5dcb5fbd2fca7842ed.log:2022-09-08T04:36:38.967438664+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-339ccff5fa5ca83dc83037db9a9a7a2751eda23c1f8774aa7b6482c36d025bdd.log:2022-09-08T04:52:38.957918194+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-39e3bb9914e0b00b9418a32b9e5f509b34f15a03f84ec3581518c1e4f599c450.log:2022-09-08T04:42:00.335207457+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-3fd1d7c09e3293bdecfe0d0304e6bac03297cf917bf5d1af5f33eabfe6e52acf.log:2022-09-08T04:20:01.176086008+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-600fb2a8e399189efe13b084306752a2c8690bdefdbb3b4cab50bbcd75c7e860.log:2022-09-08T04:26:06.533517279+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-6392e4b47975f0f481ad86c93ea5a0eff2be557280d0d1b97802c9ca6d5f8020.log:2022-09-08T04:19:34.809171691+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-648275726ac9ff8f907c13005b8014681919fe43fd651e088f4e8466e264b347.log:2022-09-08T05:08:36.308439358+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-835a6f41306a20e39fb39cecc79061ec0649ebfcbb2d41bd7b53c08ccff571d7.log:2022-09-08T04:20:36.923380469+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-9dcadbc390e798fa06441727f76913c6db14ddd0e3b034d6d53e36961e2d1dff.log:2022-09-08T04:58:06.264445401+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-a3c861c827749fad6b6e317dacc1a9cdc690488e0d78798ee33da91ecc0c49ea.log:2022-09-08T05:03:25.849682633+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-a417ba792572c730f4177c56d014eb5cfceb6a3121e97d2fbf6ee2607fb188c2.log:2022-09-08T04:47:17.746158952+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-a6672a96035c5287e5d69cc6fe41f64645a962b0cf0b5a2a7f4340c4f5ba2622.log:2022-09-08T04:19:24.267118917+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-bfeed7f8b65b694f56dcc3902f0621b76f14c46d87b6224f0146439e94534f62.log:2022-09-08T04:21:34.069784661+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-c702eed48aee4176f88b1b73eab45d27e8b7b6afb8ab132a8df5183728fb5f43.log:2022-09-08T04:31:24.54245379+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querycoord-fdb5687fd-76slv_milvus-ci_querycoord-de0dcb3576725e6900e365dd1ee545546b2ceab9256da86df4a5e00bde5d649e.log:2022-09-08T04:23:06.746508201+08:00 stderr F panic: ShowPartitions failed: can't find collection: search_collection_MYknSZ7N
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-565f8_milvus-ci_querynode-0e82082a2d2e55c1c534d2fe5f7c7c46a0f0d1b29ba5ed1439c8bcfff1ee803a.log:2022-09-08T05:10:04.494498528+08:00 stderr F panic: rpc error: code = Canceled desc = latest balancer error: last connection error: connection error: desc = "transport: Error while dialing dial tcp: lookup mdk-4-n-etcd on 10.255.0.10:53: server misbehaving"
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-565f8_milvus-ci_querynode-ddbfd4f8d95f9d9331da82e4d024ec059de59788456073dc34e5e0e1955d47b4.log:2022-09-08T03:06:48.509556822+08:00 stderr F panic: All attempts results:
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-28fc743da6052473aaefa70507687989af67cf3f7e808ee38b63b9f1d8a5b5fe.log:2022-09-08T03:06:51.640139786+08:00 stderr F panic: All attempts results:
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.2828105+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.578136298+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.57817014+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.578218898+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.583917843+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.585952389+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.592278373+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.599885966+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.607878034+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.616042994+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.623808265+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.631703403+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.639538255+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.647502049+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.655547351+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.663428571+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.671292537+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.679260988+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.687289657+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.69514798+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.703025648+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.710868005+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.718805208+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.726806199+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.734962201+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.747207478+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.75520584+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.767995538+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.775572123+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.783424079+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.791348001+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.799408898+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-a5e9805dab6d5b76a55cd5c9802f88d397b8c7613f0d991d20693321fc0b9bd1.log:2022-09-08T04:19:09.807132113+08:00 stderr F 	/usr/local/go/src/runtime/panic.go:992 +0x71
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-d0cd0f5a549ae4633a836928f8d55bdc6438cbedd3c2c797acd587c31a68b207.log:2022-09-08T05:10:14.556033987+08:00 stdout F [2022/09/07 21:10:14.555 +00:00] [ERROR] [querynode/shard_segment_detector.go:132] ["failed to handle watch segment error, panic"] [error="rpc error: code = Canceled desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing dial tcp: lookup mdk-4-n-etcd on 10.255.0.10:53: server misbehaving\""] [stack="github.com/milvus-io/milvus/internal/querynode.(*etcdShardSegmentDetector).watch\n\t/go/src/github.com/milvus-io/milvus/internal/querynode/shard_segment_detector.go:132"]
ci.var.log.containers.mdk-4-n-milvus-querynode-884d49796-dqnf7_milvus-ci_querynode-d0cd0f5a549ae4633a836928f8d55bdc6438cbedd3c2c797acd587c31a68b207.log:2022-09-08T05:10:14.559217812+08:00 stderr F panic: rpc error: code = Canceled desc = latest balancer error: last connection error: connection error: desc = "transport: Error while dialing dial tcp: lookup mdk-4-n-etcd on 10.255.0.10:53: server misbehaving"
```

### Anything else?

_No response_

--- a/ci/jenkins/Nightly.groovy
+++ b/ci/jenkins/Nightly.groovy
@@ -3,10 +3,10 @@
 // When scheduling a job that gets automatically triggered by changes,
 // you need to include a [cronjob] tag within the commit message.
 String cron_timezone = 'TZ=Asia/Shanghai'
-String cron_string = BRANCH_NAME == "master" ? "50 22,4 * * * " : ""
+String cron_string = BRANCH_NAME == "master" ? "50 22,5 * * * " : ""
 
 // Make timeout 4 hours so that we can run two nightly during the ci
-int total_timeout_minutes = 6 * 60
+int total_timeout_minutes = 7 * 60
 def imageTag=''
 def chart_version='3.1.11'
 pipeline {
@@ -183,11 +183,13 @@ pipeline {
                                                 if ("${MILVUS_SERVER_TYPE}" == "distributed-pulsar") {
                                                     clusterEnabled = "true"
                                                     tag="L0 L1 L2 ClusterOnly"
+                                                    e2e_timeout_seconds = 6 * 60 * 60
                                     
                                                 } else if("${MILVUS_SERVER_TYPE}" == "distributed-kafka" ) {
                                                     clusterEnabled = "true"
                                                     mqMode='kafka'
                                                     tag="L0 L1 L2 ClusterOnly"
+                                                    e2e_timeout_seconds = 6 * 60 * 60
                                                 }
                                                 if ("${MILVUS_CLIENT}" == "pymilvus") {
                                                     sh """ 

